styles.css

```diff
@@ -0,0 +1,704 @@
+
+/* Mobile responsiveness */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+}
+
+#navbar {
+  display: flex;
+  background-color: #3e3c3c;
+  align-items: center;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.no-style-deco {
+  list-style: none;
+}
+
+.white {
+  color: #fff;
+}
+
+#home {
+  background-color: #1c1a19;
+  background-attachment: scroll;
+  background-repeat: no-repeat;
+}
+
+.orange-color {
+  color: #f55800;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+}
+
+.project-item {
+  background-image: url("media/Img Placeholder.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.title {
+  font-family: "crete Round", sans-serif;
+  font-weight: 400;
+  color: #fff;
+}
+
+.art-btn {
+  background-color: #ff6b00;
+  border: inherit;
+  color: #fff;
+}
+
+#about {
+  background: #1c1a19;
+}
+
+@media only screen and (max-width: 768px) {
+  /* Nav bar styling */
+  #navbar {
+    padding: 0 9.3vw 0 6.4vw;
+  }
+
+  #desk-nav {
+    display: none;
+  }
+
+  #logo {
+    width: 10%;
+    height: auto;
+    border-radius: 50%;
+  }
+
+  aside {
+    order: 2;
+  }
+
+  /* Home section styling */
+  #home {
+    background-image: url("media/illustration-header-mobile.png");
+    background-size: 25%;
+    height: 25%;
+    background-position: right bottom;
+    display: flex;
+    flex-direction: column;
+    padding: 32vw 6.4vw;
+  }
+
+  .orange-color {
+    font-size: 14vw;
+  }
+
+  .project-item p,
+  .project-item h2,
+  .project-item .btn {
+    margin-right: 3.5vw;
+    margin-left: 3.5vw;
+  }
+
+  .project-item p {
+    width: 80vw;
+  }
+
+  #home p {
+    font-weight: 400;
+    font-family: 'poppins', sans-serif;
+    font-size: 4.3vw;
+    margin: 0;
+  }
+
+  #home h2 {
+    font-family: 'roboto', sans-serif;
+    font-weight: 700;
+    font-size: 6.4vw;
+    color: #fff5e1;
+    margin: 3.2vw 0;
+  }
+
+  /* Social media links styling */
+  #social-items {
+    display: flex;
+    align-self: center;
+    margin-top: 3.2vw;
+    padding: 0;
+  }
+
+  .social-item {
+    margin: 0 4vw;
+  }
+
+  /* Styling works section */
+  #projects {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    padding: 32vw 6.4vw;
+  }
+
+  #work-title {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 10.7vw;
+    font-weight: 400;
+  }
+
+  .line {
+    border: 0.3vw solid #212121;
+    width: 100%;
+    margin: 6.4vw 0 21.3vw 0;
+  }
```

Comment:
I suggest that the div element be styled using the background color attribute and giving a height to the div of 1px.